### PR TITLE
Add infobip as dynamic channel type and Update it to use the latest I…

### PR DESCRIPTION
**Pull request by norkans7**
…nfobip API

### CODE REVIEW

File: temba/channels/handlers.py [hunk 1]
```diff
@@ -684,12 +684,15 @@ class InfobipHandler(BaseChannelHandler):
     url = r'^infobip/(?P<action>sent|delivered|failed|received)/(?P<uuid>[a-z0-9\-]+)/?$'
```
Comment (nicpottier) [inline temba/channels/handlers.py]: We should move this into the /channels/infobip world.

File: temba/channels/tests.py [hunk 2]
```diff
@@ -3127,39 +3127,6 @@ def test_m3tech(self):
...
-        config_url = reverse('channels.channel_configuration', args=[channel.pk])
-        self.assertRedirect(response, config_url)
-
-        response = self.client.get(config_url)
-        self.assertEquals(200, response.status_code)
-
-        self.assertContains(response, reverse('handlers.infobip_handler', args=['received', channel.uuid]))
-        self.assertContains(response, reverse('handlers.infobip_handler', args=['delivered', channel.uuid]))
-
     @override_settings(SEND_EMAILS=True)
```
Comment (nicpottier) [inline temba/channels/tests.py]: Same here.

File: temba/channels/handlers.py (hunk 1 above)
Comment (nicpottier) [inline temba/channels/handlers.py]: Sorry, was confused, thought this was already supported. I think Rowan is going to add support for that.

File: temba/channels/types/infobip/views.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                number = '+' + number
+
+            try:
+                cleaned = phonenumbers.parse(number, None)
+                return phonenumbers.format_number(cleaned, phonenumbers.PhoneNumberFormat.E164)
+            except Exception:  # pragma: needs cover
+                raise forms.ValidationError(
+                    _("Invalid phone number, please include the country code. ex: +250788123123"))
+
+    title = _("Connect Infobip")
```
Comment (rowanseymour) [inline temba/channels/types/infobip/views.py]: Don't need any of these besides `form_class` - `ClaimViewMixin.Form` takes care of everything else

**Pull request merged**